frontend/language/src/nb.json

```
@@ -648,7 +648,7 @@
   "process_editor.not_found_process_error_message": "Det finnes ingen prosess definert innenfor BPMN, Du kan verifisere om prosessen finnes i BPMN-filen.",
   "process_editor.not_found_process_heading": "Ingen tilgjengelig prosess",
   "process_editor.save": "Lagre",
-  "process_editor.saved_successfully": "Bpmn prosessen ble lagret",
+  "process_editor.saved_successfully": "Bpmn-prosessen ble lagret",
```

Kanskje det er bedre å ha det i presens?
```suggestion
  "process_editor.saved_successfully": "Bpmn-prosessen er lagret.",
```